app/assets/stylesheets/pages/idea.sass

```diff
@@ -18,11 +18,17 @@
   .idea-title
     min-height: 100px
   .idea-actions
+    display: flex
```

Comment:
This is to allow the absolutely positioned idea-actions div to stretch to fill the entire width of the idea card. 

Looking at our [site use by browser version](https://analytics.google.com/analytics/web/#report/visitors-browser/a74251859w112373789p117321493/%3Fexplorer-table.secSegmentId%3Danalytics.browserVersion%26explorer-table.plotKeys%3D%5B%5D%26explorer-table.rowStart%3D0%26explorer-table.rowCount%3D100/) only 0.17% of site traffic uses IE < v11, and 0.06% use Firefox < v22. Do you think that affecting ~1% of users is sufficiently small to justify using Flexbox? For me, it offers a more elegant alternative to absolute positioning, and cuts a decent chunk out the time it takes me to style elements. Let me know what you think!